pkg/util/iptables/iptables.go

```diff
@@ -405,15 +405,15 @@ func (runner *runner) restoreInternal(args []string, data []byte, flush FlushFla
 		trace.Step("Locks grabbed")
 		defer func(locker iptablesLocker) {
 			if err := locker.Close(); err != nil {
-				klog.Errorf("Failed to close iptables locks: %v", err)
+				klog.ErrorS(err, "Failed to close iptables locks")
 			}
 		}(locker)
 	}
 
 	// run the command and return the output or an error including the output and error
 	fullArgs := append(runner.restoreWaitFlag, args...)
 	iptablesRestoreCmd := iptablesRestoreCommand(runner.protocol)
-	klog.V(4).Infof("running %s %v", iptablesRestoreCmd, fullArgs)
+	klog.V(4).InfoS("running command", "iptablesRestoreCmd", iptablesRestoreCmd, "fullArgs", fullArgs)
```

Comment:
```suggestion
	klog.V(4).InfoS("Running restore command", "command", iptablesRestoreCmd, "args", fullArgs)
```

Reply:
@shubheksha probably meant "Running iptables command" here not "Running restore command", which would be inconsistent with all the other messages